.tools/weathertop/backend/src/main/java/com/weathertop/lambda/HandleEventBridgeScheduler.java

```diff
@@ -0,0 +1,66 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.weathertop.lambda;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.LambdaLogger;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import com.weathertop.service.EventBridgeScheduler;
+import java.util.Map;
+
+public class HandleEventBridgeScheduler implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+
+    @Override
+    public APIGatewayProxyResponseEvent handleRequest(APIGatewayProxyRequestEvent event, Context context) {
+        LambdaLogger logger = context.getLogger();
+        EventBridgeScheduler schedule = new EventBridgeScheduler();
+        try {
+            // Extract "language" from query parameters
+            Map<String, String> queryParams = event.getQueryStringParameters();
+            if (queryParams == null ) {
+                return createErrorResponse(400, "Missing required query parameters");
+            }
+
+            String taskDefinitionArnVal = queryParams.get("taskDefinitionArnVal").toLowerCase(); // Normalize
+            String clusterName = queryParams.get("clusterName").toLowerCase();
```

Comment:
The variable clusterName is forced to lower case, but then it is compared to values that are not all lower case, like "MyNetWeathertopCluster". How does this ever match?